CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # [Unreleased](https://github.com/purecloudlabs/genesys-cloud-streaming-client/compare/v17.2.1...HEAD)
+### Fixed
+* [no-jira] Update ws to address Snyk vulnerability
```

Comment:
Can we create a quick ticket for this? It's good to have in case security ever comes knocking and I personally like to limit the amount of `[no-jira]`'s we have in the changelog.

Reply:
Sure! I had been debating if it was worth it. 😄